iota-constants/lib.rs

```diff
@@ -132,6 +155,7 @@ pub const PRIVATE_KEY_REUSE_ERROR: &str = "Private key reuse detect!";
 /// SEND_TO_INPUTS_ERROR
 pub const SEND_TO_INPUTS_ERROR: &str = "Send to inputs!";
 
+/// SecurityLevel of IOTA signature. 2 is used mostly.
```

Comment:
The rest of these comments are fine, but this one needs to be more in-depth. Say the consequences of the different security levels.